pkg/cable/vxlan/vxlan.go

```diff
@@ -304,7 +307,14 @@ func (v *vxlan) ConnectToEndpoint(endpointInfo *natdiscovery.NATEndpointInfo) (s
 		return endpointInfo.UseIP, fmt.Errorf("failed to add remoteIP %q to the forwarding database", remoteIP)
 	}
 
-	err = v.vxlanIface.AddRoute(allowedIPs, remoteVtepIP, v.vxlanIface.vtepIP)
+	cniIface, err := cni.Discover(v.localCluster.Spec.ClusterCIDR[0])
+
+	if err != nil {
+		return endpointInfo.UseIP, fmt.Errorf("failed to get the CNI interface IP for cluster CIDR %q",
```

Comment:
Recently we noticed some network-plugins not creating CNI interfaces (with an IP that belongs to the PodCIDR) on the nodes. 

In such situations is it possible to log a WARNING while continuing to support the inter-cluster traffic without the host-network support?

Reply:
Now using vxlan interface ip , if CNI interface ip does not exist.